packages/panda-san/src/navbar/navbar.scss

```diff
@@ -0,0 +1,107 @@
+.navbar {
+  align-items: center;
+  background-color: $navbar;
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  left: 0;
+  position: absolute;
+  width: 100vw;
+  z-index: 100;
```

Comment:
Can we create a `z-index` series of 'levels'? That way we can easily track what should be on top of what. We don't want to arbitrarily put any z index.